Yoast/wpseo-woocommerce
# AddEditor event listener now checks if excerpt editor is the one being added.

## Summary

This PR can be summarized in the following changelog entry:

* Fixes the Product editor crashing when loading a Product with the text editor active, and switching to the visual editor.

## Relevant technical choices:

* I forgot to check which editor was added when listening for `addEditor` events. The code in question should only fire when the excerpt editor is added, not the 'main' post editor at the top of the page. This PR fixes this.

## Test instructions

This PR can be tested by following these steps:

* Add a new WooCommerce Product.
* Open the browser's development console.
* Switch the main post editor to the text editor.
* Reload the page.
* [ ] The console should not give an error.
* [ ] The main editor should continue to behave properly.

Fixes #303
